.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 orbs:
   # Using 1.0 of the Orbs means it will use the latest 1.0.x version from https://github.com/wordpress-mobile/circleci-orbs
-  android: wordpress-mobile/android@1.0
+  android: wordpress-mobile/android@dev:feature/test-with-multiple-devices
```

Comment:
This needs to be restored to `@1.0` once the orb changes in https://github.com/wordpress-mobile/circleci-orbs/pull/61 are published.